From ce3a380dddd0cb16cb3d8d947b69657d7646c121 Mon Sep 17 00:00:00 2001
From: Wei Yongjun <yongjun_wei@trendmicro.com.cn>
Date: Wed, 13 Jul 2016 12:46:57 +0000
Subject: [PATCH] net: ethernet: bgmac: Remove redundant dev_err call in
 bgmac_probe()

There is a error message within devm_ioremap_resource
already, so remove the dev_err call to avoid redundant
error message.

Signed-off-by: Wei Yongjun <yongjun_wei@trendmicro.com.cn>
Signed-off-by: David S. Miller <davem@davemloft.net>
---
 drivers/net/ethernet/broadcom/bgmac-platform.c | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

--- a/drivers/net/ethernet/broadcom/bgmac-platform.c
+++ b/drivers/net/ethernet/broadcom/bgmac-platform.c
@@ -129,10 +129,8 @@ static int bgmac_probe(struct platform_d
 	}
 
 	bgmac->plat.base = devm_ioremap_resource(&pdev->dev, regs);
-	if (IS_ERR(bgmac->plat.base)) {
-		dev_err(&pdev->dev, "Unable to map base resource\n");
+	if (IS_ERR(bgmac->plat.base))
 		return PTR_ERR(bgmac->plat.base);
-	}
 
 	regs = platform_get_resource_byname(pdev, IORESOURCE_MEM, "idm_base");
 	if (!regs) {
@@ -141,10 +139,8 @@ static int bgmac_probe(struct platform_d
 	}
 
 	bgmac->plat.idm_base = devm_ioremap_resource(&pdev->dev, regs);
-	if (IS_ERR(bgmac->plat.idm_base)) {
-		dev_err(&pdev->dev, "Unable to map idm resource\n");
+	if (IS_ERR(bgmac->plat.idm_base))
 		return PTR_ERR(bgmac->plat.idm_base);
-	}
 
 	bgmac->read = platform_bgmac_read;
 	bgmac->write = platform_bgmac_write;
